examples/powertools-examples-parameters/src/main/java/org/demo/parameters/ParametersFunction.java

```diff
@@ -31,15 +31,28 @@
 import java.util.stream.Collectors;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
-import software.amazon.lambda.powertools.parameters.ParamManager;
-import software.amazon.lambda.powertools.parameters.SSMProvider;
-import software.amazon.lambda.powertools.parameters.SecretsProvider;
+import software.amazon.lambda.powertools.parameters.secrets.SecretsParam;
+import software.amazon.lambda.powertools.parameters.secrets.SecretsProvider;
+import software.amazon.lambda.powertools.parameters.ssm.SSMParam;
+import software.amazon.lambda.powertools.parameters.ssm.SSMProvider;
 
 public class ParametersFunction implements RequestHandler<APIGatewayProxyRequestEvent, APIGatewayProxyResponseEvent> {
     private final static Logger log = LogManager.getLogger(ParametersFunction.class);
 
-    SSMProvider ssmProvider = ParamManager.getSsmProvider();
-    SecretsProvider secretsProvider = ParamManager.getSecretsProvider();
+    // Annotation-style injection from secrets manager
+    @SecretsParam(key = "/powertools-java/userpwd")
+    String secretParamInjected;
+
+    // Annotation-style injection from Systems Manager
+    @SSMParam(key = "/powertools-java/sample/simplekey")
+    String ssmParamInjected;
+
+    SSMProvider ssmProvider = SSMProvider
```

Comment:
looks weird  `.builder().build()` could we do like the SDK have a `.create()` that does the same?

(same for other providers)

Reply:
I honestly think `build` is more common? Maybe the issue is that we're using the builder without providing anything?

Reply:
Seems reasonable. Implementing!